test/execution/target_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Wanda.Execution.TargetTest do
+  use ExUnit.Case
+
+  import Wanda.Factory
+
+  alias Wanda.Execution.Target
+
+  describe "target" do
+    test "should load checks properly when the targets have the same checks" do
```

Comment:
super-mega-nitpick: i would rename this to "return", since there is not really a "load" side effect.

Reply:
too much copy/paste XD